label_studio/core/all_urls.json

```diff
@@ -203,6 +203,12 @@
         "name": "projects:api:project-tasks-list",
         "decorators": ""
     },
+    {
+        "url": "/api/projects/<int:pk>/gt-tasks/",
```

Comment:
nit: can we name this "groundtruth-tasks" to make sure it's clear to people unfamiliar w/ our use case what this is for?

Reply:
Sure thing can change